service_auth/test/resources/evolutions/test/1.sql

```diff
@@ -55,6 +56,6 @@ INSERT INTO MUSARK_AUTH.MUSEUM_COLLECTION VALUES ('ba3d4d30-810b-4c07-81b3-37751
 INSERT INTO MUSARK_AUTH.MUSEUM_COLLECTION VALUES ('ef4dc066-b6f8-4155-89f8-7aa9aeeb2dc4', 'Marine evertebrater', '[10]');
 INSERT INTO MUSARK_AUTH.MUSEUM_COLLECTION VALUES ('c39703f4-c64c-4c13-b92c-67e6af68f674', 'ArkEtno', '[1,2]');
 
-INSERT INTO MUSARK_AUTH.AUTH_GROUP VALUES ('7963557f-e36b-4f12-951d-daa36b3840e1', 'test_test', 20, 99, 'Foo bar baz');
+INSERT INTO MUSARK_AUTH.AUTH_GROUP VALUES ('7963557f-e36b-4f12-951d-daa36b3840e1', 'test_test', 1, 20, 99, 'Foo bar baz');
```

Comment:
Just a thought, so we don't forget. We should probably provide a SQL script that updates all the _currently_ registered users to have access to storage facility. Unless the user has super user access, where we should insert the value for the `All` "module".